backend/contracts/Ticket.sol

```diff
@@ -99,6 +99,26 @@ contract Ticket is ERC721, AccessControl {
         }
     }
 
+    function getSaleStartTimePerRank()
+        external
+        view
+        returns (uint256[] memory)
+    {
+        return saleStartTimePerRank;
+    }
```

Comment:
Did you try running `npm run prettier`? I've configured this action to keep code clean 🛀